tests/test_xet_upload.py

```diff
@@ -136,6 +136,21 @@ def test_upload_file_with_bytesio(self, api, tmp_path, repo_url):
             downloaded_content = f.read()
             assert downloaded_content == self.bin_content
 
+    def test_upload_file_with_byte_array(self, api, tmp_path, repo_url):
+        repo_id = repo_url.repo_id
+        content = bytes(self.bin_content)
```

Comment:
```suggestion
        content = self.bin_content
```

no need to convert to `bytes` (it's already the case)